card_db_src/cards_db.json

```diff
@@ -1346,6 +1346,7 @@
         "randomizer": false,
         "types": [
             "Action",
+            "Liaison",
```

Comment:
Oops, I didn't mean to commit this in this PR. I may do some cleanup of these types later. I'm removing it with a force-push.